CMakeLists.txt

```diff
@@ -23,6 +23,25 @@ if(NOT OPENCL_ROOT)
     message(FATAL_ERROR "Can't find opencl")
 endif()
 
+find_path(HCC_ROOT bin/hcc
+    PATH_SUFFIXES
+        hcc
+    PATHS
+        /opt/rocm
+)
+if(NOT HCC_ROOT)
+    message(FATAL_ERROR "Can't find hcc")
+endif()
+
+set(CLANG_ROOT "${HCC_ROOT}")
+set(BITCODE_DIR "${HCC_ROOT}/lib")
```

Comment:
I thought the bitcode came from rocm-device-libs? Shouldn't this call `find_package(AMDDeviceLibsConfig)` and then `set(BITCODE_DIR ${AMD_DEVICE_LIBS_PREFIX})`?

If hcc always installs a copy of the device libs then I guess its easier to just use hcc's copy.

Reply:
Right, I tried that at first, but it led me to the lib directory for /opt/rocm/hcc/lib. If I try this on device-libs, then it will need to be /opt/rocm/lib/bitcode. It seems hcc doesn't have subdir bitcode

Reply:
Also HCC always installs its own copy of device-libs

Reply:
Well if hcc always installs a copy, then we can just use hcc's copy.